### PR TITLE
test: save original aws resource quotas as returned by API

### PR DESCRIPTION
**Pull request by richardcase**
Signed-off-by: Richard Case <richard.case@outlook.com>

**What type of PR is this?**
/kind support

**What this PR does / why we need it**:

This adds saving of the original quotas from AWS.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**:

- [x] squashed commits
- [ ] includes documentation
- [ ] adds unit tests
- [x] adds or updates e2e tests

### CODE REVIEW

File: test/e2e/shared/aws.go [hunk 1]
```diff
@@ -798,24 +798,27 @@ type ServiceQuota struct {
...
 
+	originalQuotas := map[string]*servicequotas.ServiceQuota{}
+
 	for k, v := range limitedResources {
 		out, err := serviceQuotasClient.GetServiceQuota(&servicequotas.GetServiceQuotaInput{
 			QuotaCode:   aws.String(v.QuotaCode),
 			ServiceCode: aws.String(v.ServiceCode),
 		})
 		Expect(err).NotTo(HaveOccurred())
+		originalQuotas[k] = out.Quota
```
Comment (Ankitasw) [inline test/e2e/shared/aws.go:813]: How about we only save `value` field from below:

```
classiclb:
  Adjustable: true
  ErrorReason: null
  GlobalQuota: false
  Period: null
  QuotaArn: arn:aws:servicequotas:us-west-2:705628986737:elasticloadbalancing/L-E9E9831D
  QuotaCode: L-E9E9831D
  QuotaName: Classic Load Balancers per Region
  ServiceCode: elasticloadbalancing
  ServiceName: Elastic Load Balancing (ELB)
  Unit: None
  UsageMetric:
    MetricDimensions:
      Class: None
      Resource: ClassicLoadBalancersPerRegion
      Service: Elastic Load Balancing
      Type: Resource
    MetricName: ResourceCount
    MetricNamespace: AWS/Usage
    MetricStatisticRecommendation: Maximum
  Value: 20
```

Comment (richardcase) [reply test/e2e/shared/aws.go:813]: I did think about that but thought there was no harm in saving everything...just in case.

**Pull request merged**